src/format.unit.ts

```diff
@@ -62,4 +62,38 @@ describe("format", () => {
 		form.Text = "label2";
 		expect(error.message).toBe("label2 must be formatted as #,###.");
 	});
+
+	test("lists", async () => {
+		var model = new Model({
+			"Form": {
+				Table: {
+					label: "table",
+					type: "Form.Table[]",
+					format: "[Text2]",
+					default: () => [{
+						Text1: "Text1",
+						Text2: "Text2"
+					},
+					{
+						Text1: "Text1",
+						Text2: "Text2"
+					}]
+				}
+			},
+			"Form.Table": {
+				Text1: {
+					label: "Text1",
+					default: "Text1",
```

Comment:
The defaults for `Text1` and `Text2` do not seem to be relevant for the test. The `Form.Table` type definition could likely be simplified to the following:
```ts
"Form.Table": {
    Text1: String,
    Text2: String
}
```